iina/PlayerCore.swift

```diff
@@ -612,6 +632,16 @@ class PlayerCore: NSObject {
     }
   }
 
+  func playbackRestarted() {
+    DispatchQueue.main.async {
+      Timer.scheduledTimer(timeInterval: TimeInterval(0.2), target: self, selector: #selector(self.reEnableOSDAfterFileLoading), userInfo: nil, repeats: false)
```

Comment:
mpv will send track switched event and property changed event for all observed properties when started file. There's no way to identify them as far as I know.